cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ReplicationController
 metadata: 
   name: monitoring-heapster-v4
-  namespace: default
+  namespace: system
```

Comment:
"k8s" is too obscure for users.  "kube" is better.  "kube-system" or
"kube-cluster" or "kube-infra".  How about "kube-system"?

On Thu, Jun 25, 2015 at 5:55 AM, Brendan Burns <notifications@github.com>
wrote:

> In cluster/addons/cluster-monitoring/google/heapster-controller.yaml
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10331#discussion_r33250726>
> :
>
> > @@ -2,7 +2,7 @@ apiVersion: v1
> >  kind: ReplicationController
> >  metadata:
> >    name: monitoring-heapster-v4
> > -  namespace: default
> > +  namespace: system
>
> I don't care much, but I prefer descriptive names. I agree there could be
> namespace clashes. 'k8s-system'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/10331/files#r33250726>
> .
>